components/UnorderedList/src/stories/bem.stories.mdx

```diff
@@ -47,3 +47,22 @@ import "../index.scss";
   </Story>
   {/* eslint-enable react/no-unknown-property */}
 </Canvas>
+
+### Unordered list + Paragraph
+
+<Canvas>
+  {/* eslint-disable react/no-unknown-property */}
+  <Story name="Unordered list + Paragraph">
+    <ul class="denhaag-ordered-list">
```

Comment:
I think you meant: `<ul class="denhaag-unordered-list"> `
And if this is not the situation, than you still can use this class extending it with `@extend .denhaag-ordered-list` in the scss file.

Reply:
Changed the `denhaag-ordered-list` to `denhaag-unordered-list`. and the styles are already extended ;-)